questionnaire_versions.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_dynamodb_table" "author_questionnaire_versions_table" {
+  name           = "${var.env}-questionnaire-versions"
+  hash_key       = "id"
+  billing_mode   = "PAY_PER_REQUEST"
+
```

Comment:
Does this not define a range key so to keep the incremental versions?

Reply:
Not yet. Its being worked on at the moment.

Reply:
I need to confirm that we can retrospectively add a range key to our tables.

This answer on SO suggests that it might be possible to do so using global secondary indexes. So I want to test this out by modifying the terraform config.
https://stackoverflow.com/questions/24358412/dynamodb-change-range-key-column

Found this SO post which gives a good overview of the differences between range keys and global secondary indexes.
https://stackoverflow.com/questions/23276462/dynamodb-range-vs-global-secondary-index